sr_unix/dlopen_libyottadb.c

```diff
@@ -58,18 +58,18 @@ int dlopen_libgtmshr(int argc, char **argv, char **envp, char *main_func)
 			GTM_DIST_PATH_MAX);
 		return ERR_DISTPATHMAX;
 	}
-	memcpy(&gtmshr_file[0], fptr, dir_len);
-	gtmshr_file[dir_len] = DIR_SEPARATOR;
-	memcpy(&gtmshr_file[dir_len + 1], GTMSHR_IMAGE_NAME, STR_LIT_LEN(GTMSHR_IMAGE_NAME));
-	gtmshr_file[dir_len + STR_LIT_LEN(GTMSHR_IMAGE_NAME) + 1] = 0;
+	memcpy(&yottadb_file[0], fptr, dir_len);
+	yottadb_file[dir_len] = DIR_SEPARATOR;
+	memcpy(&yottadb_file[dir_len + 1], YOTTADB_IMAGE_NAME, STR_LIT_LEN(YOTTADB_IMAGE_NAME));
+	yottadb_file[dir_len + STR_LIT_LEN(YOTTADB_IMAGE_NAME) + 1] = 0;
 
 	/* RTLD_NOW - resolve immediately so we know errors sooner than later
-	 * RTLD_GLOBAL - make all exported symbols from gtmshr available for subsequent dlopen.
+	 * RTLD_GLOBAL - make all exported symbols from libyottadb.so available for subsequent dlopen.
 	 */
-	handle = dlopen(&gtmshr_file[0], (RTLD_NOW | RTLD_GLOBAL));
+	handle = dlopen(&yottadb_file[0], (RTLD_NOW | RTLD_GLOBAL));
 	if (NULL == handle)
 	{
-		FPRINTF(stderr, "%%GTM-E-DLLNOOPEN, Failed to load external dynamic library %s\n", gtmshr_file);
+		FPRINTF(stderr, "%%GTM-E-DLLNOOPEN, Failed to load external dynamic library %s\n", yottadb_file);
```

Comment:
Leaving this as a hard-coded GTM error message?

Reply:
Good catch. Now fixed all occurrences in this module.